README.md

```diff
@@ -22,6 +22,7 @@ Below is a table of available configurations:
 | ------------------- | -------------------------- | -------------------- | ------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
 | force               | CONFIG_FORCE               | -force               | true                | overwrite secrets when not match                                                                                                  |
 | debug               | CONFIG_DEBUG               | -debug               | false               | show DEBUG logs                                                                                                                   |
+| managedonly         | CONFIG_MANAGEDONLY         | -managedonly         | false               | only modify secrets which were created by imagepullsecret                                                                         |
```

Comment:
I would have defaulted to this behavior, but AFAICT that would change existing behavior in some edge cases, so I propose here that the default doesn't alter the behavior.